examples/clamptoground.html

```diff
@@ -0,0 +1,115 @@
+
+            itowns.Fetcher.json('/examples/layers/JSONLayers/IGN_MNT.json').then(result => globeView.addLayer(result));
+            itowns.Fetcher.json('/examples/layers/JSONLayers/IGN_MNT_HIGHRES.json').then(result => globeView.addLayer(result));
+
+            menuGlobe.addGUI('RealisticLighting', false,
+                (newValue) => { globeView.setRealisticLightingOn(newValue); });
+
+            globeView.addEventListener(itowns.GLOBE_VIEW_EVENTS.GLOBE_INITIALIZED, () => {
+                // eslint-disable-next-line no-console
+                console.info('Globe initialized');
+                menuGlobe.addImageryLayersGUI(globeView.getLayers(l => l.type === 'color'));
+                menuGlobe.addElevationLayersGUI(globeView.getLayers(l => l.type === 'elevation'));
+            });
+
+            window.globeView = globeView;
+
+            const contour = new itowns.THREE.Group();
+
+            const l = new itowns.GeometryLayer('outline');
+            l.type = 'geometry';
+            l.preUpdate = (context, layer, sources) => {
+                return contour.children || [];
+            };
+            l.update = (context, layer, line) => {
+                itowns.placeObjectOnGround(globeView.referenceCrs, globeView.wgs84TileLayer, globeView.wgs84TileLayer.level0Nodes, line);
+            };
+            itowns.View.prototype.addLayer.call(globeView, l);
+
+            for (const pt of [[4, 45], [5, 48]]) {
+                var group = new itowns.THREE.Group();
+
+                // red cylinder above the ground
+                var geometry = new itowns.THREE.BoxGeometry( 10, 10, 500);
+                var material = new itowns.THREE.MeshBasicMaterial( {color: 0xff0000} );
+                var cylinder = new itowns.THREE.Mesh( geometry, material );
+                cylinder.frustumCulled = false;
+                cylinder.position.z = 250;
+                cylinder.updateMatrix();
+                cylinder.layers.set(l.threejsLayer);
+                group.add(cylinder);
+
+                // black cylinder under the ground
+                material = new itowns.THREE.MeshBasicMaterial( {color: 0x000000} );
+                cylinder = new itowns.THREE.Mesh( geometry, material );
+                cylinder.position.z = -250;
+                cylinder.frustumCulled = false;
+                cylinder.layers.set(l.threejsLayer);
+                cylinder.updateMatrix();
+                group.add(cylinder);
+
+
+                const p = new itowns.Coordinates('EPSG:4326', pt[0], pt[1]).as('EPSG:4978').xyz();
+                group.lookAt(p);
+                group.position.copy(p);
+                group.updateMatrix();
+
+                group.updateMatrixWorld(true);
+                contour.add(group);
+            }
+
+            globeView.scene.add(contour);
+            globeView.controls.setCameraTargetGeoPosition({longitude:4, latitude: 45}).then(() => {
```

Comment:
Hmm ok.
I don't really understand why (looks like a bug to me) but it's not important for this PR so I'll change this.